hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -250,6 +251,30 @@ public static void assertRowsEventuallyInAnyOrder(
             List<Object> arguments,
             Collection<Row> expectedRows,
             long timeoutForNextMs
+    ) {
+        assertRowsEventuallyInAnyOrder(sql, arguments, expectedRows, timeoutForNextMs, noop());
+    }
+
+    /**
+     * Execute a query and wait for the results to contain all the {@code
+     * expectedRows}. Suitable for streaming queries that don't terminate, but
+     * return a deterministic set of rows. Rows can arrive in any order.
+     * <p>
+     * After all expected rows are received, the method further waits a little
+     * more if any extra rows are received, and fails, if they are.
+     *
+     * @param sql          The query
+     * @param arguments    The query arguments
+     * @param expectedRows Expected rows
+     * @param timeoutForNextMs The number of ms to wait for more rows after all the
```

Comment:
missing `@param afterQueryExecuted`

Reply:
I've removed this method completely after Frantisek's comment